google/cloud/spanner_v1/instance.py

```diff
@@ -428,6 +429,7 @@ def database(
         pool=None,
         logger=None,
         encryption_config=None,
+        database_dialect=DatabaseDialect.DATABASE_DIALECT_UNSPECIFIED
```

Comment:
Docstrings for the new argument should be added

Reply:
Added